syntax = "proto2";

import "Device.proto";

package jungi.mobilus_gtw_client.proto;
option optimize_for = LITE_RUNTIME;

message DevicesListResponse {
  repeated Device devices = 1;
}